BlueDBOnDisk/src/main/java/org/bluedb/disk/segment/SegmentBatch.java

```diff
@@ -57,7 +59,7 @@
 
 	protected static Range getLargestEmptyRange(List<Range> optionsSmallToBig, Set<Range> existingChunkRanges) {
 		return optionsSmallToBig.stream()
-				.sorted(Comparator.reverseOrder())
+				.sorted(reverseOrderBySize)

```

Comment:
I'm curious why you had to make this change. Were `optionsSmallToBig` coming in the wrong order? Could something else be depending on that order as well? Should we fix the sorting on the `optionsSmallToBig` list?